tests/cases/given-names/inline.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="../../jats.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<?xml-model href="../../../src/support.sch" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<?expected-error message="<bold> in <given-names> is ignored." node="//bold"?>
+<?expected-error message="<italic> in <given-names> is ignored." node="//italic"?>
+<?expected-error message="<sub> in <given-names> is ignored." node="//sub"?>
+<?expected-error message="<sup> in <given-names> is ignored." node="//sup"?>
```

Comment:
Should we support inline stuff here? Bit more complicated to ignore them, but more complicated ones (eg `<inline-media>`!) would probably break things.

Reply:
Are these purely inline HTML element or are there ones that have no identical equivalent? (never heard of `<inline-media>` in HTML)

Reply:
Content model at https://jats.nlm.nih.gov/archiving/tag-library/1.2/element/given-names.html.

There's also the nesting problem (eg you can include [`<related-article>`](https://jats.nlm.nih.gov/archiving/tag-library/1.2/element/related-article.html), in which you can include all sorts of things...)

Reply:
Going to simplify matters and ignore elements like that in these situations, but not basic HTML. Might not present entirely as expected but 🤷‍♂.